premake5.lua

```diff
@@ -330,6 +330,7 @@ function plugincommon()
             "gcc_s",
             "gcc",
             "dl",
+            "X11",
```

Comment:
This should be in the pkg-config lines, not as library to link against directly.
Put it as lowercase "x11" at the end of the --cflags and --libs

Reply:
Thanks will adjust.